pytest_structlog/__init__.py

```diff
@@ -1,108 +1,110 @@
 import logging
```

Comment:
This needed to get moved to a subfolder to make `py.typed` adding work

Reply:
You will need to modify the `setup.py` also.

Reply:
Yup, done that I think :)

Reply:
Will also need to remove the Python :: 2 classifier, the "universal" wheel metadata, and add `python_requires=">=3.x"` for whatever minimum minor x you use in the typing additions (I don't know which). The package needs to "unadvertise" python 2 support.

Reply:
@wimglenn I'm assuming from that you want to remove 2.7 support? Just wanted to check, as redoing this so typing support doesn't block 2.7 support is doable, it's just more work!

Reply:
Yes, it can drop 2.7. But the package metadata for the next release needs to be constrained, so that older runtimes will continue to install [0.6](https://pypi.org/project/pytest-structlog/0.6/) and not attempt to install the (incompatible) typed version.